crates/test-programs/wasi-tests/src/bin/poll_oneoff_files.rs

```diff
@@ -261,7 +261,7 @@ unsafe fn test_poll_oneoff(dir_fd: wasi::Fd) {
     test_sleep();
     test_empty_poll();
     test_fd_readwrite_valid_fd(dir_fd);
-    test_fd_readwrite_invalid_fd();
+    //test_fd_readwrite_invalid_fd(); // FIXME: This traps now, rather than failing
```

Comment:
Is this still broken? Just want to note it for a follow-up so we don't forget to go back and fix it.

Reply:
Earlier I had assumed it was failing due to the issue with calling `get_stdout` multiple times, but it turns out to have been a bug in the poll_oneoff adapter. I've now pushed a patch fixing it.